group-aws/integration.md

```diff
@@ -3,6 +3,19 @@
 There are very few integration tests for EC2. Those that
 exist are in test/integration/targets
 
+## Policy
+
+Following the 2.4 release, all modules marked `stableinterface`
+MUST have integration tests for new features. Upgrading to boto3
+shall be considered a feature request.
+
```

Comment:
SHOULD/MUST all new modules be written to use boto3?

Reply:
yes, all new modules are to be boto3.  existing modules are case by case

Reply:
Migrations are happening, but no set roadmap for doing so.

Soon :tm: